### ISSUE TITLE
General Health Pages main content frontend analysis

### ISSUE THREAD

**Issue by ambikaroos** [department-of-veterans-affairs/va.gov-team#5804]
The main text above and below the alert boxes on VA.gov health pages (Prescriptions, Messaging, Test Results, Medical Records, Scheduling) needs to change depending on whether or not a user is logged in. And if a user is logged in, the text may need to change depending on whether or not they are patients at a Cerner facility. Main text will need to have two versions, for:

1. Unauthenticated users + Authenticated users who are not patients at a Cerner facility
2. Authenticated users who are patients at a Cerner facility

**Comment by ambikaroos** [comment]
Note that we'll also need the ability to determine which Cerner facility a user is a patient at: Columbus, OH (April 10, Scheduling only), Mann Grandstaff (March 28), and others in the future

**Comment by hinzed1127** [comment]
After going back and forth with different folks on the VSA/VSP/CMS teams, here's the high-level situation of where we landed:

All of those pages have their have their controlled by the Drupal CMS. Because these content updates are dependent on knowing whether or not a patient is authenticated (and whether they're a patient at a Cerner facility), it's not something supported via the CMS alone (the CMS is all static content, and this would require javascript that leverages data in the front-end's redux store).

These constraints narrow the possibilities down to a few options:

1. Adding HTML ids and class names to the Drupal markdown templates that would display specific content for authenticated/unauthenticated users. `vets-website` code would get injected into those pages and conditionally hide/display the correct content based on if a user is logged in and a Cerner patient. 

    The drawback is this adds lots of implicit logic inside of templates that ideally should only be dealing with static content. It also wouldn't necessarily scale well on its own (since it would be dependent on adding those unique ids/classes inside any templates that require this kind of conditional content). This would probably be a future question/piece of functionality for the CMS team to explore and see if it's worth implementing.

2. Move the content for these 5 pages entirely into `vets-website`. Then there's no weird divide in the conditional logic between the CMS and the react app for something that naturally can be handled by the react side. It would result in some very small and simple React apps, but it's a much more natural solution than adding a lot of undue complexity to the relationship between `vets-website` and the CMS.

   This would require a two-phased approach for updating the website: port the existing content over to react while removing it from the CMS, deploying those changes together, and **then** making the further authenticated/unauthenticated updates.

The consensus seems to be option 2, so that's what we're moving forward with.

**tl;dr** move the content for these pages from the CMS to `vets-website`, then add in the authenticated/unauthenticated content states

**Comment by ambikaroos** [comment]
Closing in favor of implementation ticket #6642
